src/mbgl/util/logging.cpp

```diff
@@ -11,21 +13,53 @@ namespace mbgl {
 namespace {
 
 static std::unique_ptr<Log::Observer> currentObserver;
+static bool useThread = true;
```

Comment:
`static` not needed in the nameless namespace

Reply:
Removed `static` from all variables